Tests/Sources/Core/TweakManagerTests.swift

```diff
@@ -88,23 +89,7 @@ class TweakManagerTests: XCTestCase {
         NotificationCenter.default.post(name: TweakProviderDidChangeNotification, object: self, userInfo: userInfo)
         XCTAssertFalse(didCallClosure)
     }
-    
-    func testTweakManagerDecryption() throws {
```

Comment:
are we sure we want to remove this test?

Reply:
Previously this test was using "LocalTweaks_example.json" file from DempApp resources, updated the test.